documents/policies/internal/Working Group Formation.md

```diff
@@ -0,0 +1,42 @@
+# WCA Working Group Formation Policy
+
+### Version 1.0 {.version}
+
+## Purpose
+The purpose of this policy is to provide guidelines for the formation and operation of working groups within the World Cube Association (WCA) in order to facilitate collaboration, innovation, and progress in specific focus areas.
+
+## Policy
+1. Definition of a Working Group
+   1. A working group is a temporary team of WCA Staff and/or community members established to address a specific project or area of interest within the organization. Working groups are tasked with generating ideas, providing recommendations, and executing initiatives related to their assigned focus area
```

Comment:
Definition is missing *why* you should set up a working group. At the moment it just seems like another way for the Board to micromanage and restrict the work and discussions WCA Staff can do/have. 

Maybe people will be willing to go through the "hoops" of this governance if there are clearer benefits to the "costs" (the months required to get board pre-approval and the ongoing reporting. Those benefits to me are having a formal process and timelines, being able to use WCA resources in the project, getting meaningful engagement from relevant WCA teams, and having a far more likely chance of having any proposed change approved by having gone through the official process. Presumably the policy should note these benefits, otherwise it is a whole bunch of compliance where it isn't readily apparent why you'd want to do a working group "officially".

Reply:
I am not sure how much of that needs to be baked into the policy. 

The why of working groups should be done in communications. If you are not doing your work in the formalised working group you risk your work not being recognised. As an organisation we want to ensure that we are focusing on the right things at the right time. 

Standardising how working groups are formed, how they are structured and operate and how report their progress should become common practice and it should become an unquestioned truth that if you want to get something done right, you do it within a formalised structure. We need to set it up in beginning to be transparent and for there to be accountability to management (whatever that looks like) and to the whole community. 

There is a whole body of literature on this topic and I don't know that it belongs in the policy.

Reply:
Fine. I'd still like to have seen something in a policy but it isn't a hill I will die on. Just as with RO's the motions have the rights (i.e. the benefits you get and why you might want to become an RO) as well as the Duties (what you need to do for that recognition). This just has the "what you need to do" and doesn't commit in policy why you'd want to do that. Of course, things like the WCRP or the Regulations are much the same :/